src/app/content/components/Page/PageNotFound.tsx

```diff
@@ -31,6 +31,10 @@ const PageNotFoundText = styled.div`
   span {
     margin-right: 0.5rem;
   }
+
+  ${SidebarControl} {
+    order: initial;
```

Comment:
instead of setting the order in the control, and overriding it here, it'd be better to only set it where its needed